protected/humhub/modules/space/views/space/about.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+use yii\helpers\Html;
+
+/* @var $space humhub\modules\space\models\Space */
+
+?>
+<div class="panel panel-default">
+    <div class="container"><h1></h1><?= Yii::t('SpaceModule.profile', '<strong>About</strong> <b>{name}</b>', ['name' => $space->name]) ?></h1></div>
+    <div class="container"><?= $space->description; ?></div>
+    <br>
```

Comment:
This is where it gets interesting, do you want to just add all the discussed content and we can discuss the layout?

Reply:
Sadly this is as far as I got with implementing the about section, I wasn't able to do much after :sweat_smile:

Reply:
There is of course also the note here https://github.com/humhub/humhub/pull/4376#issuecomment-692447815 where both User Title and Space description are connected for some reason and removing the line with `$subTitle` not only removes the space description from the space but also the title from the user profile :thinking:

Reply:
Here's the code I used when adding badges to the space directory. Maybe the same checks can be used here, and similar ones for space visibility.
```
<?php if ($space->join_policy == Space::JOIN_POLICY_NONE) {?>
                <span class="label" style="background-color:#045701;vertical-align:2px" aria-hidden="true" data-toggle="tooltip" data-placement="top" title="Invite-only">
                        <?= Yii::t('ContentModule.widgets_views_label', 'Private&nbsp;'); ?>
                                <i class="fa fa-circle main-color"></i></span>
        <?php } elseif ($space->join_policy == Space::JOIN_POLICY_APPLICATION) { ?>
                <span class="label" style="background-color:#045701;vertical-align:2px"aria-hidden="true" data-toggle="tooltip" data-placement="top" title="You can request, but you have to be approved by an admin." >
                        <?= Yii::t('ContentModule.widgets_views_label', 'Closed&nbsp;'); ?>
                                <i class="far fa-dot-circle-o main-color"></i></span>
         <?php } else { ?>
                <span class="label" style="background-color:#a9b300;vertical-align:2px" data-toggle="tooltip"  data-placement="top" title="Anyone can join">
                        <?= Yii::t('ContentModule.widgets_views_label', 'Open&nbsp;'); ?>
                        <i class="far fa-circle main-color"></i></span>
    <?php } ?>

```

Buddha also made an interesting suggestion, about showing where the logged in user is a member or not and what permissions they have.

Reply:
@Carl2213 well I went the simple route when doing this, in a similar way to how the profile About section is setup, if the user is a guest it throws a Forbidden error, really don't need to do any checks for private spaces because by default you shouldn't be able to see it because it would ask you by default to login and a private space will by default throw a space invisible error message, the same goes for the about section, the issue at hand would be implementing a custom field that would be display Owner/Admins within the space about section as well as some cleaning up of the code, everything else is pretty much glued together after that, I don't believe in complicating something that shouldn't be complicated :man_shrugging:

![Screenshot_1](https://user-images.githubusercontent.com/35392110/93290403-a4d4c880-f7ae-11ea-8de3-0b4188c10c2a.png)